src/components/Candidate.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import styled from 'styled-components';
+import { url } from './url';
+import axios from 'axios';
+
+const Candidate = ({ candidateName, voteCount, id, getCandidates }) => {
+  const getVote = async (id) => {
+    try {
+      const vote = await axios.get(`${url}/vote`, {
```

Comment:
vote가 할당되고 쓰이지 않고 있어요 없애도 될 것 같습니당

```suggestion
      await axios.get(`${url}/vote`, {
```